products/views.py

```diff
@@ -1,3 +1,68 @@
-from django.shortcuts import render
+from enum import Enum
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Q, F, Max, Min, Count
+
+from products.models import Product
+
+class BidTypeEnum(Enum):
+    BUY      = 1
+    SELL     = 2
+    END      = 3
+    CANCELED = 4
+
+class ProductListView(View):
+    def get(self, request):
+        category          = request.GET.get('category')
+        search            = request.GET.get('search')
+        shoe_size_list    = request.GET.getlist('shoe_size')
+        apparel_size_list = request.GET.getlist('apparel_size')
+        price_filter      = request.GET.get('price')
+        sort              = request.GET.get('sort')
+
```

Comment:
```suggestion
        offset =
        limit. =
```
DB에 존재하는 데이터가 많지 않다 하더라도 offset, limit를 적용해주세요!

Reply:
리뷰해주신대로 수정했습니다